tasks/main.yml

```diff
@@ -1,12 +1,27 @@
 ---
 # tasks file for roles/cli-utils
 
+- name: Import a key for epel
+  ansible.builtin.rpm_key:
+    state: present
+    key:  https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-9
+
+- name: epel | setup dnf repository
+  become: true
+  ansible.builtin.dnf:
+    update_cache: true
+    name:
+      https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+    state: present
+
 - name: system packages | cli user utils
   become: true
-  yum:
+  ansible.builtin.dnf:
+    update_cache: true
     name:
       - bzip2
       - screen
       - tmux
       - zsh
     state: present
+  when: ansible_os_family == 'RedHat'
```

Comment:
Arguably all the tasks above only applies to RedHat as well. Given that the scope of the role is EL9 only, I think we don't need to be pedantic. This can be revisited if this ever gets updated to work on Ubuntu